_includes/_code/lakes-interactive/lib/border-star-button.dart

```diff
@@ -0,0 +1,153 @@
+import 'package:flutter/material.dart';
+// Uncomment lines 3 and 6 to view the visual layout at runtime.
+//import 'package:flutter/rendering.dart' show debugPaintSizeEnabled;
+
+void main() {
+  //debugPaintSizeEnabled = true;
+  runApp(new MyApp());
+}
+
+class FavoriteWidget extends StatefulWidget {
+  @override
+  FavoriteWidgetState createState() => new FavoriteWidgetState();
+}
+
+class FavoriteWidgetState extends State<FavoriteWidget> {
```

Comment:
I suggested not using private here, for snippet simplicity purposes. It makes for easier to read code. The assumption is: if someone was making a < 100 LOC Flutter app, they wouldn't use private.

The idea was to convey that Dart and Flutter aren't ceremonial, any more than you need them to be. You can "upgrade" to library privacy when you need to, but for very small apps you probably don't need it. To show that Dart and Flutter are _scalable_ systems is important: use the ceremony when you need it, not before.